CMakeSettings.json

```diff
@@ -15,7 +15,9 @@
           "value": "True",
           "type": "BOOL"
         }
-      ]
+      ],
+      "cmakeToolchain": "C:/Users/Felipe Wannmacher/source/repos/vcpkg/scripts/buildsystems/vcpkg.cmake",

```

Comment:
This probably won't work for most users :)

Reply:
oops xD